django/contrib/admin/templatetags/admin_static.py

```diff
@@ -1,17 +0,0 @@
-from django.apps import apps
```

Comment:
I would put this on the same deprecation timeline as the staticfiles "static" tag. While it's not documented, I think many people overriding admin templates are using it, so I think it would cause a lot of pain for users trying to support multiple versions of Django if we removed it.